### PR TITLE
Update to with-supertokens example app

### PR DESCRIPTION
**Pull request by rishabhpoddar**
Updates example app code to work with latest sdk version

### CODE REVIEW

**Review by balazsorban44 — APPROVED** [review]

**Review by leerob — APPROVED** [review]
LGTM

File: examples/with-supertokens/pages/index.tsx [hunk 1]
```diff
@@ -134,6 +130,13 @@ function ProtectedPage({ userId }) {
           </div>
         </div>
 
+        <p className={styles.description}>
+          Access token payload:
+          <pre className={styles.code}>
+            {JSON.stringify(session.accessTokenPayload, null, 2)}
+          </pre>
+        </p>
+
```
Comment (leerob) [inline examples/with-supertokens/pages/index.tsx:139]:
```suggestion
```

**Comment by leerob** [comment]
Could you merge in latest please?

**Comment by rishabhpoddar** [comment]
> Could you merge in latest please?

Done @leerob

**Review by leerob — APPROVED** [review]

**Pull request merged**